src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs

```diff
@@ -28,7 +28,7 @@ partial class ListViewBase
 		public event DragItemsStartingEventHandler DragItemsStarting;
 		public event TypedEventHandler<ListViewBase, DragItemsCompletedEventArgs> DragItemsCompleted;
 
-		protected bool _isProcessingReorder;
+		private bool _isProcessingReorder;
```

Comment:
Unrelated to your changes. I see that this field is only accessed on `ListView` due to:

https://github.com/unoplatform/uno/blob/685f9c31d00b9adee8d6cb43aeebb2b4601c85e9/src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs#L238

Is this specific only to `ListView? Is it not applicable to `GridView`?

Reply:
D'oh ... you're right ! Going to check for `ListViewBase` instead ^^